modules/policy-criteria.adoc

```diff
@@ -387,16 +388,53 @@ SYS_TIME +
 SYS_TTY_CONFIG +
 SYSLOG +
 WAKE_ALARM +
-| AND, OR
+| AND
 | *Deploy*, +
 *Runtime* (when used with a Runtime criterion)
 
 
 | Add Capabilities
-| Linux capabilities that must not be added to the container, for instance the ability to send raw packets or override file permissions.
+| Linux capabilities that must not be added to the container, such as the ability to send raw packets or override file permissions. Provides alerts when the specified capabilities are added. For example, if configured with `NET_ADMIN` or `NET_RAW`, and the deployment manifest YAML file includes at least one of these two capabilities, the alert occurs. The alert would not occur if `DROP_ALL` were configured, because neither of the specified capabilities are added.
```

Comment:
Please remove this line: "The alert would not occur if `DROP_ALL` were configured, because neither of the specified capabilities are added"